services/auth-service/src/main/java/com/sourcefuse/jarc/services/authservice/enums/UserStatus.java

```diff
@@ -0,0 +1,15 @@
+package com.sourcefuse.jarc.services.authservice.enums;
+
+public enum UserStatus {
+  REGISTERED(0),
```

Comment:
no need of numbers

Reply:
I guess we store numbers in DB hence to map on the basis for the same

Reply:
Done